### ISSUE TITLE
Error trying to play from YouTube Site

### ISSUE THREAD

**Issue by elkosupertech** [smplayer-dev/smtube#31]
**Describe the bug**
I have changed the site under settings > view to YouTube.  It loaded YouTube as expected, however clicking on the video to play yields an error message from YouTube:  An Error occurred.  Please try again later.  (Playback ID: 0ciPZ-giny_lUY1q)

**To Reproduce**
1. View > Settings and changed Site to YouTube.
2. Signed into my YouTube account
3. Clicked on Video
4. Video starts to load in SMTube window the Error is displayed

**Expected behavior**
Load video into SMPlayer

**Screenshots**
![image](https://github.com/smplayer-dev/smtube/assets/3150060/f30bb337-82c3-485e-9f59-ec4faef7cde3)

**Your Enviroment**
 - SMTube version: 21.10.0 (64 Bit)
 - OS: Windows 11

**Comment by cheshirecatt** [comment]
Same problem here with SMplayer 24.5.0
![imagen](https://github.com/user-attachments/assets/43da7e42-a4d0-4561-9d8c-f494ee8a65e5)

**Comment by smplayer-dev** [comment]
It was never intended for playback to be possible directly in SMTube. This is certainly caused by the outdated HTML engine. It won't be fixed.

**Comment by cheshirecatt** [comment]
> It was never intended for playback to be possible directly in SMTube. This is certainly caused by the outdated HTML engine. It won't be fixed.

Yes, but the problem is that SMTube doesn't open Youtube videos in SMPlayer, that's what should be fixed.

**Comment by smplayer-dev** [comment]
I've just tested it. I'm afraid I can't reproduce the problem. It works for me.

**Comment by cheshirecatt** [comment]
> I've just tested it. I'm afraid I can't reproduce the problem. It works for me.

I tried again and you are right it works, but you must right click and section play with SMPlayer to the video otherwise it does not work, I will try if the same thing happens on my debian computer.


https://github.com/user-attachments/assets/60d7ba61-a9ee-471b-a363-45bafd0fd3fa

**Comment by DaniilHarun** [comment]
Same behavior in Debian and Ubuntu. I use a tablet without a keyboard and I can't right click. So without a keyboard I can't use this app.

**Comment by bionade24** [comment]
Either use tonvid.com or some browser plugin that lets your play your videos in smplayer/mpv/etc.
